reference/test/matrix/sellp_kernels.cpp

```diff
@@ -195,6 +196,20 @@ TEST_F(Sellp, MovesToDense)
 }
 
 
+TEST_F(Sellp, ConvertsToCsr)
+{
+    auto csr_mtx = gko::matrix::Csr<>::create(mtx1->get_executor());
+
+    mtx1->convert_to(csr_mtx.get());
+
+    // clang-format off
+	GKO_ASSERT_MTX_NEAR(csr_mtx,
+	                    l({{1.0, 3.0, 2.0},
+	                       {0.0, 5.0, 0.0}}), 0.0);
+    // clang-format on
+}
```

Comment:
You also need to test:
- `move_to` csr (to test the `core` function)
- `count_nonzero` kernel